release-notes/opensearch-neural-search.release-notes-3.1.0.0.md

```diff
@@ -31,7 +31,8 @@ Compatible with OpenSearch 3.1.0
 - Return bad request for stats API calls with invalid stat names instead of ignoring them ([#1291](https://github.com/opensearch-project/neural-search/pull/1291))
 - Add validation for invalid nested hybrid query ([#1305](https://github.com/opensearch-project/neural-search/pull/1305))
 - Use stack to collect semantic fields to avoid stack overflow ([#1357](https://github.com/opensearch-project/neural-search/pull/1357))
-- Filter requested stats based on minimum cluster version to fix BWC tests for stats API ([#1373](https://github.com/opensearch-project/neural-search/pull/1373))
+- Filter requested stats based on minimum cluster version to fix BWC tests for stats API ([#1373](https://github.com/opensearch-project/neural-search/pull/1373)
+- Fix neural radial search serialization in multi-node clusters ([#1393](https://github.com/opensearch-project/neural-search/pull/1393)))
```

Comment:
you probably need same entry in changelog

Reply:
We might better just keep this change in 3.1.0 release note directly at this point?

Reply:
we're target to merge it along with version 3.1, I don't think we need it in scope of 3.2 which is proceed from CHANGELOG.